src/components/pages/ImageClassification/sampleData/image_endpoints.md

```
@@ -0,0 +1,1455 @@
+- multipart form request, not usual JSON
+- nested classification status in response
+  - statuses: 
+```
+    UNKNOWN = 0
+    PENDING = 1
+    RUNNING = 2
+    COMPLETED = 3
+    FAILED = 4
+```
+- example: 
+  - POST https://dev-api.datamermaid.org/v1/projects/df2bb123-5bfc-4f3f-aa88-bb102d3b2b35/classification/images/
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)